### DISCUSSION TITLE
point_0737

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#349] [Roy]
"20191110T110159_20191110T110900_T30TVK"
![image](https://user-images.githubusercontent.com/54947195/147802002-4a23c025-50ee-4ee0-8b4b-2c8177c12b40.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20191110T110159_20191110T110900_T30TVK"
![Captura](https://user-images.githubusercontent.com/86502862/153884388-d72b7544-32c4-412d-b99e-6b0d2aba2e56.JPG)

**Comment by AntoniCesar** [comment]
20181223T111451_20181223T111449_T30TVK
![Captura](https://user-images.githubusercontent.com/86502862/153887663-41f1559b-ee28-418f-b0c1-9adfb8a52b68.JPG)
